bundle/src/main/java/com/adobe/acs/commons/adobeio/core/types/ActionImpl.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.adobeio.core.types;
+
+@SuppressWarnings("WeakerAccess")
+public class ActionImpl implements Action {
```

Comment:
Who instantiates this class? If it is consumers of the application, I suspect this can just be turned into a final class named `Action` and remove the interface. Is there ever a use case for a *different* Action implementation?